### ISSUE TITLE
Badge on dropdown button hides menu

### ISSUE THREAD

**Issue by mdr1384** [picturepan2/spectre#314]
Seems there are a couple of problems with badges when applied to dropdown buttons, see this [Stackoverflow question](https://stackoverflow.com/q/46469174/1571426).  I tried messing around with the z-index on the badge, but with no success.

**Comment by picturepan2** [comment]
@mdr1384 Can you print out the HTML and your own CSS code to CodePen?

**Comment by mdr1384** [comment]
So I was trying to get a pared-down example to show you but it was not exhibiting the problem... which resulted in me doing a total de-construction of the whole CSS and building it back up from scratch, at which point the problem on our site was also fixed.  It seems to have been caused by some z-index settings that were supposed to prevent the avatar images and active buttons from showing through a slide-out panel. I changed the code to make those elements be z-index:auto rather than try to bump the z-index of the other items up. Is there some reason why the avatar and active buttons are higher z-index by default?  Anyway I do not believe there is any issue with the badges in this regard, so I am closing this question.  Thanks for taking the time to reply though.
